docker/generate_build_matrix.py

```diff
@@ -18,17 +18,17 @@
 import tabulate
 import yaml
 
-LATEST_PYTHON_VERSION = '3.10'
+LATEST_PYTHON_VERSION = '3.11'
```

Comment:
should this stay 3.10 to match pytorch latest?

Reply:
sure let's build images for python 3.11